src/github.com/getlantern/peerscanner/cf/cf.go

```diff
@@ -0,0 +1,101 @@
+// package cf provides a utility for interacting with CloudFlare
+package cf
+
+import (
+	"crypto/tls"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/getlantern/cloudflare"
+	"github.com/getlantern/golog"
+)
+
+var (
+	log = golog.LoggerFor("cf")
+)
+
+type Util struct {
+	Client *cloudflare.Client
+	domain string
+}
```

Comment:
Note that we don't bother caching records anymore.  We don't need to since the application maintains state that includes the records.